form-runner/jvm/src/main/resources/apps/fr/components/view.xsl

```diff
@@ -499,21 +499,21 @@
                       <xh:span class="navbar-toggler-icon"></xh:span>
                     </xh:button>
                     <xh:div class="collapse navbar-collapse" id="navbarSupportedContent">
-                        <xh:ul class="navbar-nav me-auto mb-2 mb-lg-0">
+                        <xh:ul class="navbar-nav ms-auto mb-2 mb-lg-0">
                             <!-- These are typically to the right -->
-                            <xh:li class="nav-item">
+                            <xh:li class="nav-item px-3">
                             <fr:language-selector/>
                             </xh:li>
-                            <xh:li>
-            <!--                        <xh:div>-->
-            <!--                            xxx: <xf:output value="fr:workflow-stage-value()"/>-->
-            <!--                        </xh:div>-->
-                            <fr:status-icons/>
-                            </xh:li>
-                            <xh:li class="nav-item">
+<!--                            <xh:li class="nav-item px-3">-->
+<!--            &lt;!&ndash;                        <xh:div>&ndash;&gt;-->
+<!--            &lt;!&ndash;                            xxx: <xf:output value="fr:workflow-stage-value()"/>&ndash;&gt;-->
+<!--            &lt;!&ndash;                        </xh:div>&ndash;&gt;-->
+<!--                            <fr:status-icons/>-->
+<!--                            </xh:li>-->
+                            <xh:li class="nav-item px-3">
                             <fr:user-nav/>
                             </xh:li>
-                            <xh:li class="nav-item">
+                            <xh:li class="nav-item px-3">
```

Comment:
To center the content vertically: `<xh:li class="nav-item px-3 d-flex align-items-center">`